### PR TITLE
Feature/stepper responsive mobile

### PR DESCRIPTION
**Pull request by ingridklepsvik**

### CODE REVIEW

File: src/components/general/Stepper/styles.less [hunk 1]
```diff
@@ -109,5 +144,38 @@
...
+    }
+
+    .progress {
+        color: var(--color-black-alt2);
+        font-weight: 300;
+
+        @media (max-width: @mobile-max-width) {
+            grid-area: progress;
+            font-size: 12px;
+            width: 32px;
```
Comment (MartinForre) [inline src/components/general/Stepper/styles.less]: grid-unit

File: src/components/general/Stepper/styles.less [hunk 2]
```diff
@@ -81,15 +113,18 @@
...
             .badge {
+                grid-area: badge;
                 display: flex;
                 align-items: center;
                 justify-content: center;
+                top: 0;
                 height: calc(var(--grid-unit) * 3px);
                 width: calc(var(--grid-unit) * 3px);
                 font-weight: 500;
                 border: 1px solid #dcdcdc;
```
Comment (MartinForre) [inline src/components/general/Stepper/styles.less]: var(--color-something)?

File: src/components/general/Stepper/styles.less [hunk 3]
```diff
@@ -1,32 +1,72 @@
+@import '~styles/variables.less';
+
 .stepper {
-    display:flex;
-    flex-direction: row;
-    align-items: center;
+    display: grid;
+    grid-template-columns: max-content max-content auto;
+    grid-template-areas: "prev next stepPane";
     border-bottom: 1px solid #ccc;
```
Comment (MartinForre) [inline src/components/general/Stepper/styles.less]: var(--color-something)?

File: src/components/general/Stepper/styles.less (hunk 1 above)
Comment (ingridklepsvik) [reply src/components/general/Stepper/styles.less]: its now calc(var(--grid-unit) * 4px);

File: src/components/general/Stepper/styles.less (hunk 2 above)
Comment (ingridklepsvik) [reply src/components/general/Stepper/styles.less]: Fixed

File: src/components/general/Stepper/styles.less (hunk 3 above)
Comment (ingridklepsvik) [reply src/components/general/Stepper/styles.less]: fixed

**Pull request merged**